### DISCUSSION TITLE
Protein inference

### DISCUSSION THREAD

**Discussion by NicoDrou** [vdemichev/DiaNN#798] [Q&A]
Dear Vadim,

FOr that, I used the uniprot database containing all human reviewed proteins and reviewed isoforms. 
I want to quantify plasma proteins with only proteotypic peptides. But then I noticed something weird. There are no proteotypic peptides detected for HSA. Peptides are always shared with one of the HSA isoforms. Therefore I tried to investigate the different protein inference modes.
I can imagine how the Isoform IDs can end up with no proteotypic peptide but I don't understand how that is possible with the Gene and the Protein Name inference. If I understood the documentation, under the protein inference set on "gene" , the proteotypic peptides should be defined considering the protein isoforms belong to the same gene. And same should apply with the Protein Name setting, Isn't it?
I enclosed here the log book as well as the main report (2 of the data files used in the search to make it uploadable) for the "gene" setting. 

[230908_Genes_report.log.txt](https://github.com/vdemichev/DiaNN/files/12575803/230908_Genes_report.log.txt)
[230908_Genes_report.csv](https://github.com/vdemichev/DiaNN/files/12575836/230908_Genes_report.csv)

Many thanks in advance for your help
Regards,
Nicolas

**Comment by vdemichev** [comment]
Hi Nicolas,

If set to 'Genes', peptide is considered by DIA-NN proteotypic if it matches a single gene.

I see that each peptide matched to albumin is matched to both P02768 and P02768-1. I cannot find P02768-1 in the isoform uniprot database, what is its gene name in the FASTAs provided? If different from ALB, this means DIA-NN will not consider any of those peptides proteotypic.

Best,
Vadim
